src/registry.ts

```diff
@@ -120,7 +120,7 @@ export class Registry {
       try {
         this.findOrCreate(f.getObjectName(), f.getObjectType()).addFile(f);
       } catch (error) {
-        this.issues.push(new Issue({file: f, message: error, key: "registry_add"}));
+        this.issues.push(new Issue({file: f, message: error && error.toString() || "", key: "registry_add"}));
```

Comment:
what does this part do?

Reply:
converting the error to string before assinging (or "" if error is null - just to be sure)

basically:

```
if (error) 
  message = error.toString()
else 
  message = ""
```

because error was an "Error" object it was implicitly casted to string in standard.ts (assinging to the local "string description") but threw an exception when calling replace() on it because its not a string and doesnt have the replace() method.

Reply:
it looks like a hack to me, using boolean operators for returning values?

I guess the ternary operator is more appropriate?
`error ? error.toString() : ""`
at least I understand what it does, as it is not javascript specifc